### ISSUE TITLE
The element is not registered on iron-form

### ISSUE THREAD

**Issue by kolarski** [expandjs/mat-editor#1]
If you place **&lt;mat-editor>** inside **&lt;form is="iron-form" id="myform">** and call `document.getElementById('myform').serialize()` content from mat-editor is not serialized with the rest of the form

**Comment by Pupix** [comment]
Thanks for finding this bug!

The main problem is that generic forms don't recognize custom elements. So far we have been able to bypass this by placing the needed inputs into the lightDOM. This is the best solution we have found so far and we are in the works of applying it to the mat-editor too.

**Why do we shy away from implementing custom fire events?**
Because this would require unnecessary coupling between the input element and the form element used. Either the input would require knowledge of the form used or the form would require knowledge of the input used, which is not what custom elements are about. They "should just work" out of the box, without having additional .knowledge about other custom elements.
